configuration.yaml

```diff
@@ -67,8 +68,8 @@ SysBio:
 TEP:
   outputBucket: gs://otar001-core/TEPs
 TargetSafety:
-  adverseEvents: https://raw.githubusercontent.com/opentargets/curation/23.02/target_safety/adverse_effects.tsv
-  safetyRisk: https://raw.githubusercontent.com/opentargets/curation/23.02/target_safety/safety_risks.tsv
+  adverseEvents: target_safety/adverse_effects.tsv
+  safetyRisk: target_safety/safety_risks.tsv
```

Comment:
Having links to versioned repos were a source of errors. The root to the repo + version had to be moved to the globals.

Reply:
I really like this, now it's much easier to migrate to newer version of curation data